pages/api/bookings/[screeningid].js

```diff
@@ -0,0 +1,36 @@
+import mongoose from "mongoose";
+import dbConnect from "../../../lib/dbConnect";
+import Screening from "/models/Booking.js";
+
+export default async function handler(req, res) {
+  // mongoose.connect(
+  //   ""
+  // );
+
+  await dbConnect();
+
+  if (req.method == "PATCH") {
```

Comment:
This API route is used to _create_ a booking in the database. By REST principles, creating something usually happens via `POST`. I would have designed the API such that `POST /api/screenings/[screeningId]/bookings` (or something like that) would be the route to create a new booking.

Note how the final part of the URL is a plural/collection resource (`bookings`) on which a `POST` usually creates an item.